src/scheduler/batch/helpers.ts

```diff
@@ -70,6 +70,8 @@ function getUnpublishedDateContexts(annualReviewStartDate: Date): DateContext[]
   ];
 
   // fill in the dates between 1 to 5 weeks from the todayDateString
+  // TODO:- refactor to use eachWeekOfInterval (see git 39e9155437946edfb6a879328656f7e5143b767b findReminderToSend.newUnpublishedObject.)
+  // TODO:- investigate I [jen] don't think this is necessary anymore.
```

Comment:
For a recurring reminder, I think this ended up being too complex. I essentially am just doing a check for "start of nth week after annual review date". The "start of week" is actually based on the day the annual review started. e.g. if annual review commences on a Wednesday, then the start of week will be Wednesday.

Reply:
Does it make sense to put a refactor ticket together?

Reply:
https://trello.com/c/xovuF5Vd/425-refactor-annual-review-batch-simplify-currentannualreview-creation